README.md

```diff
@@ -1,86 +1,126 @@
-The official Docker images for the .NET Framework on Windows Server Core.
+# Supported tags and respective `Dockerfile` links
 
-# .NET Framework Containers
+-       [`4.6.2`, `latest` (*4.6.2/Dockerfile*)](4.6.2/Dockerfile)
+-       [`3.5` (*3.5/Dockerfile*)](3.5/Dockerfile)
 
-There are multiple flavors of the .NET Framework Docker images available, for both Web and Console application workloads.
+For more information about these images and their history, please see [the relevent Dockerfile (`microsoft/dotnet-framework-docker`)](https://github.com/microsoft/dotnet-framework-docker/search?utf8=%E2%9C%93&q=FROM&type=Code). These images are updated via [pull requests to the `microsoft/dotnet-framework-docker` GitHub repo](https://github.com/dotnet/dotnet-docker/pulls?utf8=%E2%9C%93&q=).
 
-## .NET Framework 4.6.2
+[![Downloads from Docker Hub](https://img.shields.io/docker/pulls/microsoft/dotnet-framework.svg)](https://hub.docker.com/r/microsoft/dotnet-framework)
+[![Stars on Docker Hub](https://img.shields.io/docker/stars/microsoft/dotnet-framework.svg)](https://hub.docker.com/r/microsoft/dotnet-framework)
 
-The .NET Framework 4.6.2 is included in the [Windows Server Core base image](https://hub.docker.com/r/microsoft/windowsservercore/). You can pull/FROM microsoft/windowsservercore to use .NET Framework 4.6.2.
+# What is the .NET Framework?
 
-To make discovery easier, we have also provided the same image (with a FROM wrapper) in this repo, enabling you to pull `microsoft/dotnet-framework`.
+The .NET Framework is a general purpose development platform maintained by Microsoft. It is the most popular way to build client and server applications for Windows and Windows Server. It is included with Windows, Windows Server and Windows Server Core. It includes server technologies such as ASP.NET Web Forms, ASP.NET MVC and Windows Communication Framework (WCF) applications, which you can run in Docker containers.
 
-- Optional tags: `latest`, `4.6.2`
-- Dockerfile: [4.6.2/Dockerfile](https://github.com/Microsoft/dotnet-framework-docker/blob/master/4.6.2/Dockerfile)
+.NET has several capabilities that make development easier, including automatic memory management, (runtime) generic types, reflection, asynchrony, concurrency, and native interop. Millions of developers take advantage of these capabilities to efficiently build high-quality web and client applications.
 
-## .NET Framework 3.5
+You can use C#, F# and VB to write .NET Framework apps. C# is simple, powerful, type-safe, and object-oriented while retaining the expressiveness and elegance of C-style languages. F# is a multi-paradigm programming language, enabling both functional and object-oriented patterns and practices. VB is a rapid development programming language with the deepest integration between the language and Visual Studio, providing the fastest path to a working app.   
```

Comment:
Could do. I'm going to skip it for now since it's not super popular. Keep the message simple.